### ISSUE TITLE
Kee keeps prompting for basic authentication login

### ISSUE THREAD

**Issue by tomwiggers** [kee-org/browser-addon#255]
Hi,

There is a bug somewhere that if Firefox (it happens to me in FF, haven't tried other browsers), is set to reopen your tabs on launch, and you had a tab open that requires HTTP basic authentication, the Kee Vault browser plugin will keep opening prompt infinitelely until you manage to click  a login in one of the windows. I have attached a screenshot:

![2020-03-01 15_20_56-](https://user-images.githubusercontent.com/17702902/75629695-66960400-5be4-11ea-83bb-08cc6277f0be.png)

This has been a problem for a very, very long time. It would be cool if it finally gets fixed.
The upside is that it gives me a somewhat nostalgic feeling because it reminds me of when you finish a game of Microsoft Spider Solitaire.

_Originally posted by @tomwiggers in https://github.com/kee-org/browser-addon/issues/254#issuecomment-593117626_

**Comment by luckyrat** [comment]
I can't reproduce this problem. Perhaps you have another browser extension installed that modifies Firefox behaviour? Anyway, if you want to provide some more information or contribute to similar discussions, this is what the [community forum](https://forum.kee.pm) is for, rather than this code issues tracker.

**Comment by tomwiggers** [comment]
Hi,

I am re-opening this issue as this is still a problem and I have verified it is not another extension causing this, I disabled every extension except for Kee.

The repropath is as follows:

Setup:
1. Have a KeePass database opened in KeePass and have Kee enabled in Firefox and connected to the KeePass database.
2. Have the checkbox under Network Authentication named "When Kee finds only one matching network login (e.g. HTTP Auth), automatically login" **unchecked**.
3. Have Firefox set up to re-open tabs from last session when you start it again.

Repro path:
1. Open a tab which requires HTTP basic authentication and login using username/password suggested by Kee.
2. Open another tab (2) and have that tab selected.
3. Restart Firefox, the two tabs should open again with tab 2 selected.
4. Go back to tab 1 with the HTTP authentication.
5. Kee will present anywhere between a few up to practically infinite  amount of Network Authentiation windows up to the point where they don't even load fully anymore as shown in my first screenshot when opening this issue.

Thank you for reading this and I hope you're able to reproduce and solve it.
